keyboards/ymd75/config.h

```diff
@@ -23,15 +23,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #define VENDOR_ID       0x20A0
 #define PRODUCT_ID      0x422D
-#define DEVICE_VER      0x0200
 
 #define MANUFACTURER    YMDK
 #define PRODUCT         YMD75 / MT84
-#define DESCRIPTION     75% Keyboard
-
 
 #define BACKLIGHT_PIN D4
 #define BACKLIGHT_LEVELS 12
+#define RGBLIGHT_EFFECT_KNIGHT_OFFSET 4
```

Comment:
Is this applicable for all revisions, or just the newly added rev3?

Reply:
I have not tested it on previous versions. I can move it to the rev3 config.